src/main/java/com/shippo/model/Batch.java

```diff
@@ -55,6 +55,8 @@ public static enum BatchStatus {
 	@SerializedName("label_filetype")
 	private LabelFileType labelFileType;
 
+	private boolean test;
```

Comment:
I didn't see it in the api ref, but looks like it is returned in the response

Reply:
will circle back with an audit of the api ref